packages/advanced-logic/specs/payment-network-any-declarative-0.1.0.md

```diff
@@ -1,21 +1,13 @@
 # Payment Network - Any currency - Declarative
 
-You can be interested in this document if:
```

Comment:
Why are we removing these?

Reply:
I factorized it in payment-networks and referred to it.

Reply:
The file was missing, my bad. Please check again !

Reply:
The file was missing, my bad. Please check again !